chain/network/src/network_protocol/network.proto

```diff
@@ -16,8 +16,9 @@ import "google/protobuf/timestamp.proto";
 // should be in serialized form.
 // TODO: move to a separate file, probably in a separate package.
 message AccountKeyPayload {
+  reserved 1;
```

Comment:
The "old format" is actually a "never used format". If it was in use then we would need a 2-phase rollout, but here I'm just changing the proto layout a bit before the launch of the feature.

You are right that a rename of a message is wire-format-backward-compatible, but for some reason the "buf" binary that we use for checking backward compatibility is not smart enough to realize that.

It is a best practice to keep the backward-compatibility checker satisfied (even if the previous version was never used) just for sake of consistency and it is not like we are likely to run out of integers to index the fields with :).

Reply:
As I read the documentation now, "buf" binary seems to be quite opinionated on what is wire-backward-compatible, rather than to just stick to the specification, for example: https://docs.buf.build/breaking/rules#field_same_oneof
The authors even admit that they were just lazy in some aspects: https://docs.buf.build/breaking/rules#reserved_message_no_delete . I wonder now, maybe we should look for some more precise tool then :/ .